backend/data/final_data/archive/runprocessors_archive.py

```diff
@@ -44,14 +44,14 @@
     if args.stage == "all":
         # Run all the stages from top to bottom
         res = input(
-            f"Careful. You are about to run all stages INCLUDING the scrapers... Enter 'y' if you wish to proceed or 'n' to cancel: "
+            "Careful. You are about to run all stages INCLUDING the scrapers... Enter 'y' if you wish to proceed or 'n' to cancel: "
         )
         if res == "y":
-            for s in run:
-                run[s](args.year)
+            for _, stage_f in run.items():
```

Comment:
Probably for stage_f in run.values() instead

Reply:
sgtm